### PR TITLE
Email service

### PR DESCRIPTION
**Pull request by jvserivera**

### CODE REVIEW

File: src/lib/infrastructure/Services/email_service.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+export default class EmailService implements i.IEmailService {
+    
+    constructor( )
+    {
+
+    }
+
+    sendVerificationEmail(recepient:String):Promise<Object>{
+
+    const nodemailer = require('nodemailer');
```
Comment (kenneth-rosario) [inline src/lib/infrastructure/Services/email_service.ts]: use ES6 or typescript imports

File: src/lib/infrastructure/Services/email_service.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+
+    var transporter = nodemailer.createTransport({
+        service: 'gmail',
+        auth: {
+            user: 'internviewpr@gmail.com',
```
Comment (kenneth-rosario) [inline src/lib/infrastructure/Services/email_service.ts]: Move to .env.local and let us know in clickup or whatsapp of email credentials

File: src/lib/infrastructure/Services/server.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import nodemailer from 'nodemailer'
+
+
+var transporter = nodemailer.createTransport({
```
Comment (kenneth-rosario) [inline src/lib/infrastructure/Services/server.js]: Move example to tests folder in lib

**Review by kenneth-rosario — CHANGES_REQUESTED** [review]

File: src/lib/infrastructure/Services/email_service.ts (hunk 2 above)
Comment (jvserivera) [reply src/lib/infrastructure/Services/email_service.ts]: Estoy verificando porque cuando uso los credenciales guardados en mis env variables me sale este problema de autenticacion: 
    Error: connect ECONNREFUSED 127.0.0.1:587
        at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {
      errno: 'ECONNREFUSED',
      code: 'ESOCKET',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 587,
      command: 'CONN'
    }

**Pull request merged**